src/elements/common/modal.scss

```diff
@@ -6,6 +8,10 @@
         justify-content: center;
         padding: 15px 0 0;
 
+        [class^='bp_base_button_module_button']:last-child {
```

Comment:
i wonder if we should try using the Modal.Footer from blueprint to handle the buttons